arangod/RocksDBEngine/RocksDBVPackIndex.cpp

```diff
@@ -2935,7 +2937,24 @@ struct RocksDBVPackStreamIterator : AqlIndexStreamIterator {
   }
 
   LocalDocumentId load(std::span<VPackSlice> projections) const override {
```

Comment:
The `span` received by this function is a by-value parameter, and this function then modifies it.
I don't see the modified span being returned by this function, so I am wondering why it is modified in the first place.
I would understand it if the span were passed by reference and not by value, but that doesn't seem to be the case here.

Reply:
A span is just a memory region. We do not modify the span itself, but the memory it points to. It's like a string_view for vector.